system_tests/nox.py

```diff
@@ -208,4 +214,37 @@ def session_compute_engine(session):
 
 def session_app_engine(session):
     session.virtualenv = False
-    session.run('pytest', 'app_engine/test_app_engine.py')
+
+    if SKIP_GAE_TEST_ENV in os.environ:
```

Comment:
Intentional, it's a common idiom that the presence of an environment variable is treated as 'truthy'.